File: LeetCode-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> map = new HashMap<Integer, Integer>();
        for (int i = 0; i < nums.length; i++) {
            map.put(nums[i],i);
        }
        for (int j = 0; j < nums.length; j++) {
            int temp = target - nums[j];
            if (map.containsKey(temp) && j != map.get(temp)) {
                return new int[] {j,map.get(temp)};
            }
        }
        return new int[] {0};
    }
}

File: LeetCode1.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        l = len(nums)
        for x in range(l):
            for y in range(l):
                if x != y:
                    if nums[x] + nums[y] == target:
                        return x, y